### ISSUE TITLE
Correct channel names for voice assistants

### ISSUE THREAD

**Issue by StevoNL** [Sholofly/lghorizon#116]
**Is your feature request related to a problem? Please describe.**
In Home Assistant the channel names are displayed correctly. Using speech to set the input, the command is correctly executed to the correct channel, however the voice response has the leading "P" with the channel name and truncates the channel name, especially if it has a number. 
Not entirely shure if this is related to this integration or to Google Assistant, but this wonderful integration's predecessor had the option to disable the "P" and it worked correctly then.

For example with Google Nest Hub: 
![LGHorizon](https://github.com/Sholofly/lghorizon/assets/64106345/7681dc90-69a1-4afe-8e64-aa5f5034e0c6)

**Describe the solution you'd like**
If possible for the channel names not only to look right, but sound right as well.

**BTW**
This is hands down my favorite integration for Home Assistant up till date! Keep up the good work! 🥇

**Comment by Sholofly** [comment]
Where do you examine this logs?

**Comment by StevoNL** [comment]
The screenshot is from myactivity.google.com. It shows my voice prompt and the voice response from Google Assistant. It seems the channel names are passed to Google Assistant as "p NPO", "p VRT canv", "p RTL" etc.

Everything is executed perfectly by Home Assistant and there are no errors, the debug logs don't share any valuable information imo. For example: this is the log entry for a single request (key's removed):

2024-04-09 15:29:04.651 INFO (Thread-24 (_thread_main)) [lghorizon.lghorizon_api] Executing API call to https://spark-prod-nl.gnp.cloud.ziggogo.tv/eng/web/linear-service/v2/replayEvent/crid:~~2F~~2Fgn.tv~~[..........]returnLinearContent=true&language=nl

**Comment by Sholofly** [comment]
True, I don't think its an integration issue: In my account it's reporting no p:

![image](https://github.com/Sholofly/lghorizon/assets/901632/cc5e7863-97bb-4319-a176-964c59b39370)

**Comment by StevoNL** [comment]
Interesting. I'll try and remove the integration from Home Assistant and Google Home and then do a reinstall. Perhaps something is still lingering somewhere now. I'll come back with more information after I tried.

**Comment by StevoNL** [comment]
Allright, an update with high failure rate...  Things I've tried:

- Remove LGHorizon integration, restarted Home Assistant, synced devices on Google Home to make sure TV is removed. Re-added LGHorizon integration, restarted Home Assistant again and installed integration, resynced devices on Google Home:
_Problem persists_
- Tried voice promts via Google Nest mini, Google Home mini, Google Nest Hub and android Phone. Tried locally via Wifi (Local fulfillment) aswell as via 5G (Cloud fulfillment): 
_Problem persists_
- Redeployed Google Assistant testversion through actions.google.com: 
_Problem persists_

The channel names show correctly on device screens, even in Google Home, yet the voice responses remain truncated and with leading "P". 
Sholofly's reply confirm my suspicion that the cause doesn't lie in the integration. I suspect it's probably caused by a cache lingering somewhere far far away, but I'm out of clues. 

Thanks Sholofly for your quick response, if anyone else has the same problem, has an idea or know's the solution... I'd be happy to hear!
